src/field_type.cpp

```diff
@@ -139,7 +139,7 @@ void field_type::load( JsonObject &jo, const std::string & )
     optional( jo, was_loaded, "accelerated_decay", accelerated_decay, false );
     optional( jo, was_loaded, "display_items", display_items, true );
     optional( jo, was_loaded, "display_field", display_field, false );
-    optional( jo, was_loaded, "wandering_field_id", wandering_field_id, "fd_null" );
+    optional( jo, was_loaded, "wandering_field", wandering_field_id, "fd_null" );
```

Comment:
Using of `wandering_field_id` was intentional, so we could finalize later, but json node name should be changed from `wandering_field` to `wandering_field_id`.